cmake/aws-c-cal-config.cmake

```diff
@@ -1,7 +1,10 @@
 include(CMakeFindDependencyMacro)
 
+find_dependency(aws-c-common)
+
 if (NOT BYO_CRYPTO AND NOT WIN32 AND NOT APPLE)
-    find_dependency(LibCryptoCAL)
+    set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_LIST_DIR}/modules")
```

Comment:
Same question as https://github.com/awslabs/aws-c-io/pull/323#discussion_r497159053

Reply:
yup, fixed while you were reviewing